tests/test_client.py

```diff
@@ -24,6 +24,8 @@
 
 
 class TestClient(SquareClient):
+    SHIFT_MINUTES = 2
```

Comment:
Maybe let's add a comment about why we chose 2 minutes (because shifts can't overlap).